daemon/stats/collector_unix.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/opencontainers/runc/libcontainer/system"
 )
 
+/*
```

Comment:
Would it be possible to do this in `go` only ? If not, would we extract this (i.e. `getNumberOnlineCPUs`) into another package, probably in `pkg` instead ?